doc/interop-test-descriptions.md

```diff
@@ -621,7 +620,7 @@ response_type, then it should fail the RPC with INVALID_ARGUMENT.
 
 If the request sets fill_username, the server should return the client username
```

Comment:
This was added later (without my knowledge) and is a separate feature. This should be split into a different heading.